src/styles/main.scss

```
@@ -0,0 +1,76 @@
+@import "./utils/variables";
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: $height-stop-watch;
+  width: $width-stop-watch;
+  border: 5px solid black;
```

don't forget to create variables for all colors
```suggestion
  border: 5px solid $c-black;
```

---

Ок